client/components/backup-storage-space/usage-warning/use-storage-status-text.ts

```diff
@@ -0,0 +1,24 @@
+import { useTranslate, TranslateResult } from 'i18n-calypso';
+import { useMemo } from 'react';
+import { StorageUsageLevels } from '../storage-usage-levels';
+
+const useStorageStatusText = ( usageLevel: StorageUsageLevels ): TranslateResult | null => {
+	const translate = useTranslate();
+
+	// TODO: For StorageUsageLevels.Warning, estimate how many days until
+	// all storage is used, and show that in the status text.
+	return useMemo( () => {
+		switch ( usageLevel ) {
+			case StorageUsageLevels.Warning:
+				return translate( 'You will reach your storage limit soon.' );
+			case StorageUsageLevels.Critical:
+				return translate( "You're running out of storage space." );
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'You’re running out of storage space.' )` *ES Score: 7*

Reply:
This sounds like a good recommendation.